src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,112 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int CAPACITY_MULTIPLIER = 2;
+    private Node<K, V>[] table;
+    private int size;
+    private int tableLength;
+
+    public MyHashMap() {
+        table = new Node[INITIAL_CAPACITY];
+        tableLength = INITIAL_CAPACITY;
+    }
 
     @Override
     public void put(K key, V value) {
-
+        Node<K, V> newNode = new Node<>(getKeyHash(key), key, value, null);
+        size = putNodeAtPosition(newNode, getKeyHash(key) % tableLength) ? size + 1 : size;
+        if (size > tableLength * LOAD_FACTOR) {
```

Comment:
I recommend creating a variable for tableLength * LOADFACTOR - threshold. and give the first check to increase the table and then add the elements